lib/resources/cardOrders.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const ResourceBase = require('./resourceBase');
+
+class CardOrders extends ResourceBase {
+
+  constructor (config) {
+    super('cards', config);
+  }
+
+  list (cardId, params, callback) {
+    return this._transmit('GET', `${encodeURIComponent(cardId)}/orders`, null, params, null, callback);
```

Comment:
Does `encodeURIComponent` include the `/cards` part of the endpoint? Since for orders the endpoint looks like `cards/{card_id}/orders`

Reply:
Great question, `encodeURIComponent` does not. I'm not quite sure how the `cards/` part of `cards/{cardId}/orders` is being passed along into the request, but I ran the card orders tests and it seems to be hitting the correct endpoint & returning the expected data. So I didn't explicitly add it to `${encodeURIComponent(cardId)}/orders`.

In Lob-Node-Admin, [a "cards" endpoint is specified](https://github.com/lob/lob-node-admin/blob/7e606c02334577015c72fae34b2bde7072f21693/lib/resources/cards.js#L4), but I'm not sure how to add that here in Lob-Node

Reply:
this is part of the constructor for `ResourceBase`: https://github.com/lob/lob-node/blob/master/lib/resources/resourceBase.js#L9-L10